code/controllers/subsystem/processing/instruments.dm

```diff
@@ -0,0 +1,86 @@
+PROCESSING_SUBSYSTEM_DEF(instruments)
+	name = "Instruments"
+	init_order = INIT_ORDER_INSTRUMENTS
+	wait = 0.5
+	flags = SS_KEEP_TIMING
```

Comment:
Make this a mix of `SS_TICKER | SS_BACKGROUND | SS_KEEP_TIMING`, and set `wait` to `1` so its logged properly in ticks. `SS_BACKGROUND` will then also bump the priority to normal (`SS_TICKER` will treat wait as tick value, but also bump priority, so you use background to bump it out)